plugins/org.preesm.algorithm/model/Schedule.xcore

```diff
@@ -139,6 +138,7 @@ interface SequentialSchedule extends Schedule {
  * There is no hypothesis on the order on the execution of its children.
  */
 interface HierarchicalSchedule extends Schedule {
+	refers AbstractActor attachedActor
```

Comment:
Add comment on the meaning of this reference and that it can be left null when scheduling a flatten/srdag graph